02-Advanced-topics/05-GSD-visualization.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>Requiring freud and gsd is fine. But let's not install it for them. Just link to them and let the users install them in the way that they prefer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>